WooCommerce/Classes/POS/Presentation/CartView.swift

```diff
@@ -224,6 +217,29 @@ private extension CartView {
             }
         }
     }
+
+    var cartEmptyView: some View {
+        VStack {
+            Spacer()
+            // By designs, the text should be vertically centred with the image 40px above it.
+            // SwiftUI doesn't allow us to absolutely pin a view to the centre then position other views relative to it
+            // Instead, we can centre the text, and then put the image in an offset overlay.
+            Text(Localization.addItemsToCartHint)
+                .font(Constants.secondaryFont)
+                .lineSpacing(Constants.cartEmptyViewTextLineHeight)
+                .foregroundColor(Color.posTertiaryText)
+                .multilineTextAlignment(.center)
+                .overlay(alignment: .center) {
```

Comment:
Turns out dynamic type issue is easy solvable by aligning this overlay on `top`.

```suggestion
                .overlay(alignment: .top) {
```